content_schemas/dist/formats/finder/frontend/schema.json

```diff
@@ -331,6 +331,9 @@
         "generic_description": {
           "type": "boolean"
         },
+        "hide_contents_list": {
```

Comment:
As mentioned in the government-frontend PR, I'd change this to `show...` rather than `hide...` to match the `show_metadata_block` attribute.